bbb/services.py

```diff
@@ -513,8 +513,14 @@ def handlePending(self, data, msg):
         # except update our own runId. If we created a new BuildRequest for it
         # we'd end up with an extra Build.
         if our_task:
```

Comment:
As a possible scenario:
* move `our_task = ...` here
* add a write lock before querying the DB
* release the lock